### PR TITLE
AvatarToken style and doc update suggestions

### PR DESCRIPTION
**Pull request by georgewrmarshall**
Some suggestions on using BEM classNames instead of inline styles as well as some doc updates. Does not include updated tests!

### CODE REVIEW

**Comment by NidhiKJha** [comment]
Not all the changes seem relevant to [ this PR ](https://github.com/MetaMask/metamask-extension/pull/15466). Should we move the required files to a different commit?

File: ui/components/component-library/avatar-token/avatar-token.js [hunk 1]
```diff
@@ -27,52 +29,44 @@ export const AvatarToken = ({
...
+        'avatar-token',
+        showHalo && 'avatar-token--with-halo',
+        className,
       )}
       {...{ backgroundColor, borderColor, ...props }}
     >
       {showFallback ? (
-        tokenName?.[0]?.toUpperCase() ?? '?'
+        tokenName?.[0] ?? '?'
       ) : (
```
Comment (NidhiKJha) [inline ui/components/component-library/avatar-token/avatar-token.js:51]: @georgewrmarshall shouldn't we keep `tokenName?.[0]?.toUpperCase()` to keep the initial uppercase?

File: ui/components/component-library/base-avatar/base-avatar.scss [hunk 2]
```diff
@@ -28,4 +28,5 @@
   display: flex;
   align-items: center;
   justify-content: center;
+  text-transform: uppercase;
```
Comment (georgewrmarshall) [inline ui/components/component-library/base-avatar/base-avatar.scss:31]: @garrettbear mentioned that we should be changing the text to uppercase via CSS. It's more performant so we can remove changing via javascript @NidhiKJha

File: ui/components/component-library/avatar-token/avatar-token.js (hunk 1 above)
Comment (georgewrmarshall) [reply ui/components/component-library/avatar-token/avatar-token.js:51]: https://github.com/MetaMask/metamask-extension/pull/15575/files#r945938185

**Pull request closed without merging**